code/game/machinery/computer/Operating.dm

```diff
@@ -162,5 +160,24 @@
 			. = TRUE
 	. = TRUE
 
+/obj/machinery/computer/operating/multitool_act(mob/living/user, obj/item/I)
```

Comment:
This has no range check so you can link them from infinitely far away

Reply:
Oh, huh, I was thinking tool_act would handle adjacency checks. Alright.

Reply:
Actually, no other `multitool_act` implementations seem to handle range checks themselves?

Reply:
There is no range limit to linking to the computer I mean

Reply:
Ohh. Fixed.